docs/ml-guide.md

```diff
@@ -938,4 +964,4 @@ model.transform(test)
 {% endhighlight %}
 </div>
 
-</div>
\ No newline at end of file
+</div>
```

Comment:
What's the difference here?  I'm confused why this diff is showing up (but wanted to make sure it was not a weird character or something in the line.